spec/lib/mercury/cps_spec.rb

```diff
@@ -192,30 +192,30 @@ def foo
 
   it 'obeys the left identity law' do
     # return a >>= f  ===  f a
-    expect_identical(lift{a}.and_then(&method(:to_string)),
+    expect_identical(lift { a }.and_then(&method(:to_string)),
                      to_string(a),
                      '123')
   end
 
   it 'obeys the right identity law' do
     # m >>= return  === m
-    m = lift{123}
-    expect_identical(m.and_then{|x| lift{x}},
+    m = lift { 123 }
+    expect_identical(m.and_then { |x| lift { x } },
                      m,
                      123)
   end
 
   it 'obeys the associativity law' do
     # (m >>= f) >>= g  ===  m >>= (\x -> f x >>= g)
-    m = lift{123}
-    expect_identical((m.and_then(&method(:to_string))).and_then(&method(:twice)),
-                     m.and_then {|x| to_string(x).and_then(&method(:twice)) },
+    m = lift { 123 }
+    expect_identical(m.and_then(&method(:to_string)).and_then(&method(:twice)),
+                     m.and_then { |x| to_string(x).and_then(&method(:twice)) },
                      '123123')
   end
 
   def expect_identical(lhs, rhs, expected_value)
-    expect{|b| lhs.run(&b)}.to yield_with_args(expected_value)
-    expect{|b| rhs.run(&b)}.to yield_with_args(expected_value)
+    expect { |b| lhs.run(&b) }.to yield_with_args(expected_value)
+    expect { |b| rhs.run(&b) }.to yield_with_args(expected_value)
```

Comment:
`|b|` and `(&b)`, not to mention that this has to be a block in the first place, is all unfortunate noise. Why glorify it by expanding it with spaces everywhere?